### ISSUE TITLE
Can't show uncategorized torrents 

### ISSUE THREAD

**Issue by zhufeng** [rmartin16/qbittorrent-api#7]
torrents = qbt_client.torrents_info(category='null');
torrents = qbt_client.torrents_info(category='NULL');
torrents = qbt_client.torrents_info(category='');
torrents = qbt_client.torrents_info(category='Uncategorized');
torrents = qbt_client.torrents_info(category='uncategorized');

I just want to print uncategorized torrents, and I tried all above way, but none of them works.

From  qbittorrent official api guide:
> 
`Get torrents with the given category (empty string means "without category"; no "category" param means "any category")`
> 

Could you help a bit？Thanks a lot .

**Comment by rmartin16** [comment]
hmm....i may need to ask the qBittorrent developers if this still works. as you point out, their wiki says to send the empty category param with the `torrents/info` request to retrieve torrents without a category. however, when i do this, i still get back all of the torrents as well.

Torrents in qBittorrent
![image](https://user-images.githubusercontent.com/387848/77072049-71fa7500-69c3-11ea-9504-f7c978cff1b7.png)

```
>>> len(q.torrents.info(category=''))
[2020-03-19 09:16:34,139] {urllib3.connectionpool:221} DEBUG - Starting new HTTP connection (1): localhost:8079
[2020-03-19 09:16:34,140] {urllib3.connectionpool:428} DEBUG - http://localhost:8079 "GET /api/v2/torrents/info?category= HTTP/1.1" 200 803
[2020-03-19 09:16:34,140] {qbittorrentapi.request:125} DEBUG - Request URL: (GET) http://localhost:8079/api/v2/torrents/info?category=
[2020-03-19 09:16:34,140] {qbittorrentapi.request:130} DEBUG - Response status: 200 (OK)
3
>>> len(q.torrents.info(category='', sort='ratio'))
[2020-03-19 09:17:44,474] {urllib3.connectionpool:221} DEBUG - Starting new HTTP connection (1): localhost:8079
[2020-03-19 09:17:44,476] {urllib3.connectionpool:428} DEBUG - http://localhost:8079 "GET /api/v2/torrents/info?category=&sort=ratio HTTP/1.1" 200 786
[2020-03-19 09:17:44,476] {qbittorrentapi.request:125} DEBUG - Request URL: (GET) http://localhost:8079/api/v2/torrents/info?category=&sort=ratio
[2020-03-19 09:17:44,477] {qbittorrentapi.request:130} DEBUG - Response status: 200 (OK)
3
>>>
```

Let me open up a ticket with qBittorrent to see what they think.

[`torrents/info` source](https://github.com/qbittorrent/qBittorrent/blob/master/src/webui/api/torrentscontroller.cpp#L241)

**Comment by rmartin16** [comment]
quick workaround in the interim though

```python
no_cat_torrents = list(filter(lambda t: t.category=='', q.torrents.info(category='')))
```

**Comment by rmartin16** [comment]
appears to be a known issue with qBitorrent: qbittorrent/qbittorrent#11748

closing this issue since it'll be resolved once the qBittorrent team resolves the underlying issue.

**Comment by rmartin16** [comment]
they fixed qbittorrent/qBittorrent#11748.

i'll keep an eye out to see if they push it back to v4.2.

**Comment by zhufeng** [comment]
> they fixed [qbittorrent/qBittorrent#11748](https://github.com/qbittorrent/qBittorrent/issues/11748).
> 
> i'll keep an eye out to see if they push it back to v4.2.

hi there，qb 4.2.2 released, problem resolved. thank you .
